README.md

```diff
@@ -322,6 +323,25 @@ typedef NS_ENUM(NSInteger, NYTAdRequestState) {
 };
 ```
 
+## Boxing Enumerated Types
+
+Althought `enum`s are integral values, they cannot be directly used as [boxed literals](http://clang.llvm.org/docs/ObjectiveCLiterals.html#boxed-enums). So when creating a dictionary using `enum`s, they should be written as boxed literals instead of their corresponding `NSNumber` expression. 
```

Comment:
![screen shot 2015-04-22 at 14 58 41](https://cloud.githubusercontent.com/assets/102904/7282759/221e2cd2-e900-11e4-9c01-e50bbf1d1ff5.png)

I don't feel strongly either way, except to note that an enum _could_ be backed by a type which isn't strictly speaking  an `integer` in C.